analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -740,13 +764,19 @@ def signal_handler(signum, frame):
     if not os.path.exists(success_dir):
         os.makedirs(success_dir)
 
+    # Similar to failed dir, but generated both in case of success and failure.
+    reproducer_dir = os.path.join(output_path, "reproducer")
```

Comment:
If I run the analysis without the `--generate-reproducer` option it will create an empty `reproducer` directory in my report directy. I think if this option is not specified we can skip creating this directory.